util/README.md

```diff
@@ -9,6 +9,11 @@ augment or enhance default functionality.
 
 ## Features
 
+### `link`
+
+Provides implementations for parsing and formatting entities' URLs, such as
```

Comment:
Where does the term “entity” come from in this context?

Reply:
In relation to Discord's documentation it doesn't come from anywhere and is just a common term to describe an instance of a resource. I've been leaning towards the use of it in our documentation because there's not a much better way to concisely describe "something." The [work in progress cache](https://github.com/twilight-rs/cache) uses the relating terminology extensively, so I feel this would fit in with the rest of the libraries and their designs well.